projects.md

```diff
@@ -1,8 +1,42 @@
-TODO: Make this a comprehensive guide about how to find the desired project and provide an overview about all available projects/repositories
+# The different projects in the Eclipse IDE and their setup 🚀
+
+Contributing to open source can seem intimidating at first, but don’t worry! Getting involved in the Eclipse projects is actually straightforward, and you’ll be working with an awesome community.
+
+💻 **JDT** is all about the tools for developing Java within the IDE, making Java development smooth and efficient. 
+
+🔄 **EGit** provides all the tools needed to work with Git version control, essential for collaboration in open source projects. 
+
+🔌 **PDE** focuses on developing plugins for Eclipse, allowing users to extend its functionality in new and exciting ways. 
+
+🌐 **Platform** contains the core code that powers Eclipse as a Rich Client Platform (RCP), which is used to build powerful desktop applications, no matter the type of app.
```

Comment:
yes and no.. the first list gives an overview of the main acronyms and project's roles in the IDE. If we want to add details in the table, the problem is the table size, it can not be resized more than a limit and even on a wide screen it is too compact...
The best solution would be to have a nice picture of the different projects and their links together instead of this list

Reply:
Understand.
If it's a general information, should this then maybe be placed at a more general place, like https://github.com/eclipse-ide#structure-of-the-eclipse-ide-and-its-eco-system?

> The best solution would be to have a nice picture of the different projects and their links together instead of this list

If that also explains the projects then yes.

Reply:
Moved.